owncloudApp/src/main/java/com/owncloud/android/ui/activity/FileDisplayActivity.kt

```diff
@@ -1693,6 +1699,10 @@ class FileDisplayActivity : FileActivity(), FileFragment.ContainerActivity, OnEn
         else -> R.id.nav_all_files
     }
 
+    override fun optionLockSelected(type: LockType) {
+        manageOptionLockSelected(type)
+    }
```

Comment:
This method will be deleted if you apply the changes I suggested before, just like in LoginActivity.